CBLClient/Apps/CBLTestServer-Java-Desktop/build.gradle

```diff
@@ -56,8 +53,8 @@ java {
 }
 
 dependencies {
-    compile 'commons-daemon:commons-daemon:1.2.2'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation 'commons-daemon:commons-daemon:1.2.2'
+    implementation group: 'junit', name: 'junit', version: '4.12'
```

Comment:
This one should be "testImplementation 'junit:junit:4.12'"

Reply:
done